twilio/twiml/messaging_response.py

```diff
@@ -7,7 +7,7 @@ class MessagingResponse(TwiML):
     """
     def __init__(self):
         """
-        Create a new <Response>
+        Create a new <MessagingResponse>
```

Comment:
Yes you are correct, I was omitting the XML header since it's common to both classes.